index.html

```diff
@@ -11,7 +11,7 @@
 </head>
 <body>
     <header class="header-container fixed-top">
-            <!-- <div class="logo"><img src="/img/My Logo.png" alt="my logo"></div> -->
+            
             <a href="#" class="logo"><img src="/img/My Logo.png" alt="my logo"></a>
```

Comment:
[OPTIONAL] The background of the header you designed in last milestone seems to have `body's backgound` as its background. 

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221016796-b8b4f592-1c0e-42ea-9acb-a511034daa1e.png)| ![image](https://user-images.githubusercontent.com/61361037/221016350-98d8f766-b81c-4694-a291-8d077c999d25.png)|